kubernetes/cmsweb/docs/cmsweb_testbed_cluster_doc.md

```diff
@@ -0,0 +1,49 @@
+## CMSWeb Testbed Cluster
+
+There is a new testbed cluster in kuberneteds for CMSWeb in "CMS Web" project which is now available for developers to test their services. The URL to access this cluster is listed below:
+
+- https://cmsweb-k8s-testbed.cern.ch
+
+Users/developers may request for access of this cluster in their namespace in [https://gitlab.cern.ch/cms-http-group/doc/issues/199](https://gitlab.cern.ch/cms-http-group/doc/issues/199) forum. 
+
+   
+## Service Deployment Steps in K8S testbed cluster:
+
+- `ssh lxplus-cloud`
+- `export OS_PROJECT_NAME="CMS Web"`
+
+Use export KUBECONFIG to point to the relevant configurations for the clusters. Please note that these configurations are for the backend clusters as users will only need to work in the backend clusters to deploy/undeploy their services.  
+
+- `export KUBECONFIG=/eos/cmsweb/config.cmsweb-k8s-services-testbed`
+
+To see all pods in a particular namespace, following command can be used:
+   - `kubectl -n <namespace> get pods`
+
+To login to a specific pod, use:
+
+   - `kubectl -n <namespace> exec -ti <pod-name> bash`
+
+To delete a pod, use:
+   - `kubectl -n <namespace> delete pod/<pod-name>`
+
+To force delete a pod if above command does not work, use:
+   - `kubectl -n <namespace> delete pod/<pod-name> --force --grace-period=0`
+
+To deploy new service, the service first needs to be deleted and then redeployed. Following commands can be used for this purpose. 
+- Clone this repository:
+
+   - `git clone https://github.com/dmwm/CMSKubernetes.git`
+
+- Go into CMSKubernetes/kubernetes/cmsweb directory:
+
+- Update yaml file with the new version. For example open relevant service file (xxx.yaml e.g crabserver.yaml) and change version:
+   - `vim services/xxx.yaml`
+
+- Deploy newly changed yaml file. First delete old service from K8S and deploy new one. For example: 
+
+   - `kubectl delete -f services/xxx.yaml`
+   
+ - Then, following command can be used to deploy service in testbed cluster. 
```

Comment:
Please add explicitly that `#PROD#` should be replaced with **6** spaces, otherwise the markdown strip off all six spaces into one when people visually see the document. I just noticed this when I read your markdown in gist.

May be it is because you used single back-quote, instead use 3 back-quotes for command below, i..e. 3 back-quotes, then on new line put the command and then on new line put 3 back-quotes again. This would make a block in markdown to be used to declare the command.

Reply:
@vkuznet ,

I have u

> I see that you managed to make proper commit for dbs-global.yaml, right? If not I wanted to suggest that you can do these steps:
> 
> 1. rm services/dbs-global.yaml
> 2. git checkout -- services/dbs-global.yaml
>    The first step just remove the file and second step restore it as is from git repository. This was you can restore file w/o your tweaks of editor.

@vkuznet, yes, I managed to make proper commit and did exactly what you said here and changed accordingly. Now it should be fine.

Reply:
> Please add explicitly that `#PROD#` should be replaced with **6** spaces, otherwise the markdown strip off all six spaces into one when people visually see the document. I just noticed this when I read your markdown in gist.
> 
> May be it is because you used single back-quote, instead use 3 back-quotes for command below, i..e. 3 back-quotes, then on new line put the command and then on new line put 3 back-quotes again. This would make a block in markdown to be used to declare the command.

@vkuznet , I changed file according to your instructions and gist is properly displaying six spaces. I have also added comments about #PROD# string. 

Thanks

Best
M Imran